pyuvdata/uvdata/uvdata.py

```diff
@@ -4629,7 +4647,7 @@ def antpairpol_iter(self, squeeze="default"):
         for key in antpairpols:
             yield (key, self.get_data(key, squeeze=squeeze))
 
-    def conjugate_bls(self, convention="ant1<ant2", use_enu=True, uvw_tol=0.0):
+    def conjugate_bls(self, *, convention="ant1<ant2", use_enu=True, uvw_tol=0.0):
```

Comment:
yeah, I think you're right. I'll change it on that PR.

Reply:
The `order` parameter is maybe similar in the various reordering methods, should I also allow it to be positional? I think I didn't at first because I first ran into it in `reorder_blts` where there's both `order` and `minor_order`. But `minor_order` is not always required even there.

Reply:
I think it's natural to have `order` as a positional argument. `minor_order` shouldn't be.

Reply:
ok, I fixed that.